ValueChangedGenerator/ValueChangedGenerator/ValueChangedGenerator.Test/ValueChangedGeneratorUnitTests.cs

```diff
@@ -19,7 +22,7 @@ public async Task TestMethod1()
         }
 
         //Diagnostic and CodeFix both triggered and checked for
-        [TestMethod]
+        [Fact]
         public async Task TestMethod2()
```

Comment:
For your information, TestMethod2 was already broken when I forked the repo. Could you please check it when you have some time?

Reply:
Not for a while...

Reply:
❌ Not for a while
⭕ Not at all

`TestMethod1` and `TestMethod2` are default codes in the "Analyzer with Code Fix" template. These are useless for the ValueChangedGenerator.

Reply:
As far as using CSharpCodeFixVerifier, it is difficult to test code fix which generates additional C# files. This use case is out of CSharpCodeFixVerifier'2 scope. Maybe we should give up testing the Code Fix (and recommend users to migrate to the Source Generator).